queue_services/entity-emailer/src/entity_emailer/email_templates/BC-ALT-DRAFT.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <base href="/">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="referrer" content="origin-when-cross-origin"/>
+    <meta name="author" content="BC Registries and Online Services">
+    <title>How to use BCRegistry.ca</title>
+    [[style.html]]
+  </head>
+
+  <body>
+    <table class="body-table" role="presentation">
+      <tr>
+        <td>
+          [[header.html]]
+
+          <div class="container">
+            <p>Your business has been added to the BC Business Registry.</p>
+
+            [[whitespace-16px.html]]
+            [[business-info.html]]
+
+            [[whitespace-24px.html]]
+            <p>You may now file your alteration filing to become a Benefit Company.</p>
+
+            [[whitespace-16px.html]]
+            <p>To file your alteration:</p>
+            <ol>
+              <li>Log in to your BC Registries account</li>
+              <li>Click 'Open' to access <a href="{{ entity_dashboard_url }}">your business dashboard</a></li>
```

Comment:
Needs a space before "your".

Also, what are they clicking "open" on? That's an inline link right?

Reply:
Yes, the template I got underlined 'your business dashboard' but doesn't indicate anything for the 'Open'. I will ask the BA for this requirement.

Reply:
> Needs a space before "your".

Oops never mind the space. The sample looks good!